<launch>
  <arg name="launch_prefix" default="" />
  <arg name="node_namespace" default="apriltag_ros_continuous_node" />
  <arg name="camera_name" default="/camera" />
  <arg name="camera_frame" default="camera" />
  <arg name="image_topic" default="image_raw" />
  <rosparam command="load" file="$(find ground_truth)/config/settings.yaml" ns="$(arg node_namespace)" />
  <rosparam command="load" file="$(find ground_truth)/config/tags.yaml" ns="$(arg node_namespace)" />
  <node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="$(arg node_namespace)" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)" >
    <remap from="image_rect" to="$(arg camera_name)/$(arg image_topic)" />
    <remap from="camera_info" to="$(arg camera_name)/camera_info" />
    <param name="camera_frame" type="str" value="$(arg camera_frame)" />
    <param name="publish_tag_detections_image" type="bool" value="true" />
    <param name="remove_duplicates" type="bool" value="true" />
  </node>
</launch>
